cryptostore/data/parquet.py

```diff
@@ -48,10 +51,11 @@ def __init__(self, config=None):
                 self.prefix.append(config['S3']['prefix'])
                 self.kwargs.append({'creds': (config['S3']['key_id'], config['S3']['secret']), 'endpoint': config['S3'].get('endpoint')})
             if 'GD' in config:
-                self._write.append(google_drive_write)
-                self.prefix.append(config['GD']['prefix'])
+                g_drive = GDriveConnector(config['GD']['service_account'], exchanges, config['GD']['prefix'], self.default_path)
+                self._write.append(g_drive.write)
+                self.prefix.append(None)
                 self.bucket.append(None)
-                self.kwargs.append({'creds': config['GD']['service_account']})
+                self.kwargs.append({'dummy': 'Dummy'})
```

Comment:
is this needed?

Reply:
If kwargs list is not the same size than the others (func, bucket and prefix), this loop fails because of the `zip` function:
https://github.com/bmoscon/cryptostore/blob/67faea70dd9ad986eb66e2a0d3788ebc12e2dceb/cryptostore/data/parquet.py#L97


So I confirm this line has no much use, but I need it to contain the same number of elements than the others. Morevover, the added element has to be a dict so that the subsequent line does not raise error:
https://github.com/bmoscon/cryptostore/blob/67faea70dd9ad986eb66e2a0d3788ebc12e2dceb/cryptostore/data/parquet.py#L101

Reply:
ah right, ok, thats fine